.travis.yml

```diff
@@ -9,7 +9,14 @@ install:
 - bash .travis-fork-fix
 - go get github.com/mattn/goveralls
 - go get github.com/wadey/gocovmerge
-- curl https://glide.sh/get | sh
+- if [[ $(uname -m) == 's390x' ]]; then
+      wget https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-s390x.tar.gz;
```

Comment:
Please remove the unnecessary trailing semicolons on this line and the subsequent lines.

Reply:
Hi @phlogistonjohn , removing the trailing semicolons inside `if-else` condition makes the build fail with `syntax error: unexpected end of file`, hence the semicolons are required.

Reply:
That's weird but may be due to embedding the script within yaml. I'm OK with it for now, perhaps we can improve it later.